jenkins/jobs/macros-common.yaml

```diff
@@ -152,6 +152,24 @@
           pip install -r swift-uploader/requirements.txt
           swift-uploader/swiftuploader/upload.py -r DFW --password $PASSWD -c CILogs {upload_source} $LOG_PATH
 
+- builder:
+    name: osci-swift-upload-logs-with-console-vgpu
+    builders:
+      #- shell: "/usr/local/jenkins/slave_scripts/grab_console_log.sh"
+      - shell: |
+          #!/bin/sh
+          ls -l test.log logs
+          BUILD_TAG=${BUILD_TAG:-"temp_build"}
```

Comment:
{} is used by JJB to replace it with a predefined variable.
you should use {{...}} instead for {....}.